src/pages/nfts/collections/[collectionAddress].tsx

```diff
@@ -24,15 +24,13 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
   }
 
   try {
-    await store.dispatch(fetchCollection(collectionAddress))
-
-    const state = store.getState()
+    const collectionsData = await getCollections()
```

Comment:
We should also fetch the all collections to have it on swr cache, otherwise cache will be overwritten with only one collection on server side